imageregistry/v1/types_imagepruner.go

```diff
@@ -24,6 +25,7 @@ type ImagePrunerList struct {
 
 // ImagePruner is the configuration object for an image registry pruner
 // managed by the registry operator.
+// +openshift:compatibility-gen:level=1
```

Comment:
level 4, exposed=false and make a note to revisit with the image team as a followup to figure  out if users touch this.